jaxrs-api/src/main/java/javax/ws/rs/core/Response.java

```diff
@@ -1296,6 +1311,10 @@ public default Status toEnum() {
          * 307 Temporary Redirect, see {@link <a href="http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.3.8">HTTP/1.1 documentation</a>}.
          */
         TEMPORARY_REDIRECT(307, "Temporary Redirect"),
+        /**
+         * 308 Permanent Redirect, see {@link <a href="https://tools.ietf.org/html/rfc7538">HTTP/1.1 documentation</a>}.
+         */
+        PERMANENT_REDIRECT(308, "Permanent Redirect"),
```

Comment:
Not related to this PR, but still: what's the point in citing RFC 2616? It has been obsoleted years ago.